adapter/config/types.go

```diff
@@ -395,8 +408,10 @@ type APICtlUser struct {
 
 // ControlPlane struct contains configurations related to the API Manager
 type controlPlane struct {
-	Enabled                    bool
-	ServiceURL                 string `toml:"serviceUrl"`
+	Enabled    bool
+	ServiceURL string
```

Comment:
Formatting issues.

Reply:
When there is comment in the middle, this is how VSCode formats it. 
If we change it manually, then the developers would have to keep doing it repeatedly.